paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/CustomerSheetNetworkRequests.kt

```diff
@@ -51,7 +51,7 @@ internal fun billingDetailsParams(): RequestMatcher {
 internal fun fullBillingDetailsParams(): RequestMatcher {
     return RequestMatchers.composite(
         bodyPart(urlEncode("billing_details[name]"), urlEncode(CustomerSheetPage.NAME)),
-        bodyPart(urlEncode("billing_details[phone]"), CustomerSheetPage.PHONE_NUMBER),
+        bodyPart(urlEncode("billing_details[phone]"), urlEncode("+1${CustomerSheetPage.PHONE_NUMBER}")),
```

Comment:
Were we sending this to the backend incorrectly!??!?

Reply:
Unfortunately yeah :(

Reply:
Was this broken in PaymentSheet as well?

Reply:
Yeah since we weren't manually converting the text input. `InlineLinkSignup` did it properly but billing configuration did not.

Reply:
Interesting. Nice fix!